### ISSUE TITLE
Test fail on Windows

### ISSUE THREAD

**Issue by hankache** [perl6/perl6-lwp-simple#36]
```
This is Rakudo version 2019.07.1 built on MoarVM version 2019.07.1
implementing Perl 6.d.
```
OS: Windows 10
```
===> Testing: LWP::Simple:ver<0.106>:auth<github:perl6>
# Failed test 'Found pattern in downloaded file'
# at t/getstore.t line 38
# You failed 1 test of 10
===> Testing [FAIL]: LWP::Simple:ver<0.106>:auth<github:perl6>
Aborting due to test failure: LWP::Simple:ver<0.106>:auth<github:perl6> (use --force-test to override)
```

**Comment by JJ** [comment]
Thanks for the report. We'll have to add Windows testing, I guess.

**Comment by hankache** [comment]
For the record, I am not sure this issue should be raised here.
The failure only occurs when Rakudo is built on Windows 10 using  `--moar-option="--ld=link"`

**Comment by JJ** [comment]
Also, it does seem to be unrelated to the operating system. Anyway, we'll check it out.

**Comment by hankache** [comment]
Indeed, it seems this is not related to Windows. Same test also failing on WSL

**Comment by hankache** [comment]
Should be fixed by #37
